backend/app/adapter/routing/api.yml

```
@@ -15,6 +15,7 @@ servers:
   - url: https://api.short-d.com
   - url: https://api-staging.short-d.com
   - url: https://api-testing.short-d.com
+  - url: https://localhost.com
```

```suggestion
  - url: http://localhost
```
We don't use HTTPS for local servers because we are not giving developers our private certificate needed for TLS.

---

gotchya